LibreNMS/ComposerHelper.php

```diff
@@ -44,7 +44,10 @@ public static function postInstall(Event $event)
             self::setPermissions();
         }
 
+        // "optimize" doesn't clear view cache etc.
+        self::exec('php artisan optimize:clear');
         self::populateEnv();
+        self::exec('php artisan optimize');
```

Comment:
Isn't "optimize" depracated?

Reply:
it was added back again, https://github.com/laravel/framework/pull/23468